### DISCUSSION TITLE
Question: Keyboard and Mouse support simultaneously

### DISCUSSION THREAD

**Discussion by fkahhaleh** [houkanshan/arduino-bt-hid-proxy#10] [Q&A] [answered]
Hi,
I am trying to identify if this library will support both a KB and Mouse at the same time, using a single Arduino device?
The readme text indicates that in some way "_**Bridge Bluetooth keyboard and mouse to wired USB.**_ ", but quick glimpse
at the code I see Keyboard mainly.

Thanks

**Comment by houkanshan** [comment, marked as answer]
Hey thanks, this project only support keyboard. I'll correct the readme.

**Comment by fkahhaleh** [comment]
Thank you.
Perhaps Mouse can be added. Something to look forward to contribute maybe :)
